kentikapi/client.go

```diff
@@ -132,25 +128,29 @@ func (c *Config) FillDefaults() {
 		c.APIURL = APIURLUS
 	}
 
-	if c.CloudExportHostPort == "" {
-		c.CloudExportHostPort = cloudExportHostPort
-	}
-
-	if c.SyntheticsHostPort == "" {
-		c.SyntheticsHostPort = syntheticsHostPort
-	}
-
 	if c.Timeout == nil {
 		c.Timeout = pointer.ToDuration(defaultTimeout)
 	}
 
 	c.RetryCfg.FillDefaults()
 }
 
-func makeConnForGRPC(c Config, hostPort string) (grpc.ClientConnInterface, error) {
+func parseAPIURL(apiURL *string) error {
```

Comment:
```suggestion
func makeAPIV5URL(apiURL string) (string, error) {}
```
It is usually more readable to return results of operation as return function parameter instead of modifying input parameters. Also it is safer not to modify config.APIURL, which is then used to make gRPC URL, because it introduces coupling between those two pieces of logic. For example, changing hostname here would change hostname also for gRPC URL, which might not be desirable

Also, I see the naming issue repeated here. This function does more than parsing API URL. Hence its name should be different - I added suggestion.

Reply:
Done